content/downloads/devel/index.md

```diff
@@ -3,14 +3,57 @@ title = "Development builds"
 template = "devel-builds.html"
 +++
 
-## Linux
+## Installation notes
 
-TODO: list dependencies
+Development snapshot builds do not touch configuration of stable
+version; they use separate config file (`dosbox-staging-git.conf`)
```

Comment:
maybe: _Development snapshots use a separate conf file named `dosbox-staging-git.conf`, located alongside the stable conf file in:_